client/src/main/java/io/pravega/client/stream/impl/ReaderGroupStateManager.java

```diff
@@ -421,4 +414,8 @@ void checkpoint(String checkpointName, PositionInternal lastPosition) throws Rei
             throw new ReinitializationRequiredException();
         }
     }
+
+    public String refreshDelegationToken(Segment segmentId) {
```

Comment:
This is invoking `controller.getOrRefreshDelegationTokenFor`, why don't we call this method the same name to be consistent?

Reply:
Fixed.